### ISSUE TITLE
Quick advice on howto install

### ISSUE THREAD

**Issue by githubbapoopa** [akaalias/obsidian-remember-cursor#2]
Hi,
could you give me a quick advice on howto install?
I downloaded the code unzipped it.
Changed to directory and ran "npm i"

d:\______all_new______\_Daten\Obsidian\Obsidian Plugins\obsidian-recursor-master>npm i

added 47 packages, and audited 48 packages in 8s

6 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

When I then run
d:\______all_new______\_Daten\Obsidian\Obsidian Plugins\obsidian-recursor-master>npm run dev

I get these errors

> obsidian-sample-plugin@0.9.7 dev
> rollup --config rollup.config.js -w

[!] SyntaxError: Cannot use import statement outside a module
d:\______all_new______\_Daten\Obsidian\Obsidian Plugins\obsidian-recursor-master\rollup.config.js:1
import typescript from '@rollup/plugin-typescript';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .js] (d:\______all_new______\_Daten\Obsidian\Obsidian Plugins\obsidian-recursor-master\node_modules\rollup\dist\shared\loadConfigFile.js:621:13)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at loadConfigFromBundledFile (d:\______all_new______\_Daten\Obsidian\Obsidian Plugins\obsidian-recursor-master\node_modules\rollup\dist\shared\loadConfigFile.js:626:42)

Could you help me out, please?

Sorry I'm a newbie on this...

**Comment by rsyqvthv** [comment]
yes, newbie here aswell, don't know how to install it as this plugin isn't in the community plugins list. :(
